nl_server/tests/ner_place_model_test.py

```diff
@@ -22,6 +23,49 @@
 from parameterized import parameterized
 
 
+def _remove_punctuations(s):
+  s = s.replace('\'s', '')
+  s = re.sub(r'[^\w\s]', ' ', s)
+  return " ".join(s.split())
+
+
+# The function below is similar to the one under server.lib.nl.place_detection.py
+# It performs similar heuristics (punctuation removal, title casing, ending with a period)
+# which are also performed in the NL Interface.
+def _detect_places_heuristics(ner_model, query):
```

Comment:
curious, why are we duplicating it in the test? can we not factor out and call the function from util?

Reply:
There were all sorts of import errors with util being inside of lib. I'll see if I can refactor differently.